server/lib/config/pbench-server.cfg.example

```diff
@@ -38,10 +38,19 @@ port = 0000
 host = graphql.example.com
 port = 0000
 
+## User authentication section to use when authorizing the user with a Keycloak broker
 [authentication]
+
+# URL of the Keycloak auth server with port
 server_url = keycloak.example.com:0000
-realm = pbench
-client = pbench-dashboard
+
+# Realm name that is used for the authentication with keycloak
+realm = Pbench
+
+# Client entity name requesting Keycloak to authenticate a user
+client = Pbench-dashboard
+
+# Client secret if the above client is not public
 secret = client-secret
```

Comment:
Yes, it is generic to any OIDC provider. I included the term Keycloak because the Pbench server _is_ using the Keycloak broker as a front-gate even with other identity providers we may have. However, I do understand your point, if I get any other change requested to this PR I will address this along with it.

Reply:
Addressed it, I think the distinction is important.